docs/mastering-plone/registry.md

```diff
@@ -47,51 +47,53 @@ More info in {doc}`code`
 ## Introduction
 
 Do you remember the fields `audience` and `type_of_talk` in the talk content type?
-We provided several options to choose from that were hard-coded in the schema.
+The schema previously hard-coded several options for selection.
 
-Next we want to add a field to assign talks to a room.
-Since the conference next year will have different room names, these values need to be editable.
+Next, you will add a field to assign talks to a room.
+The room names change each year for the conference, so site administrators need to edit them.
 
-And while we're at it: It would be much better to have the options for `audience` and `type_of_talk` editable by admins as well, e.g. to be able to add _Lightning Talks_!
+Additionally, admins should be able to edit the options for `audience` and `type_of_talk`, making it possible to add options like _Lightning Talks_!
 
-By combining the registry, a control panel and vocabularies you can allow rooms to be editable options.
+By combining the registry, a control panel and vocabularies you can make rooms configurable options.
 
-To be able to do so you first need to get to know the registry.
+To achieve this you first need to get to know the registry.
 
-## The Registry
 
-The registry is used to get and set values stored in records.
-Each record consists of the actual value, as well as a field that describes the record in more detail.
-It has a nice dict-like API.
+## The registry
 
-Since Plone 5 all global settings are stored in the registry.
+The registry stores and retrieves values in records.
+Each record consists of the actual value, along with a field that describes the record in more detail.
+You can interact with the registry using Python dictionary-style operations to get and set values.
 
-The registry itself is provided by [plone.registry](https://pypi.org/project/plone.registry) and the UI to interact with it by [plone.app.registry](https://pypi.org/project/plone.app.registry)
+Since Plone 5 the registry stores all global settings.
+Plone provides the registry through [plone.registry](https://pypi.org/project/plone.registry) and offers a user interface for interaction via [plone.app.registry](https://pypi.org/project/plone.app.registry).
 
-Almost all settings in `/plone_control_panel` are actually stored in the registry and can be modified using its UI directly.
+Most settings in `/plone_control_panel` reside in the registry.
```

Comment:
Let's use the name that appears in the UI. I don't recall that there was ever something called `/plone_control_panel`.

```suggestion
Most settings in Site Setup reside in the registry.
```